sycl/test-e2e/Basic/host-task-dependency.cpp

```diff
@@ -178,15 +178,15 @@ int main() {
 }
 
 // launch of Gen kernel
-// CHECK:---> urKernelCreate(
```

Comment:
This is not to block the PR, but my opinion is that we shouldn't have tests relying on tracing in `e2e` tests. Those should instead be refactored to be unit-tests. I've highlighted this already in PI removal PR and I would be happy to contribute to such rewrite myself (if only I had some time for it, sigh).